securedrop_client/gui/conversation/export/device.py

```diff
@@ -122,53 +122,3 @@ def _move_export_service_to_thread(self, thread: QThread) -> None:
 
         self._export_service.moveToThread(self._export_service_thread)
         self._export_service_thread.start()
-
-
-class TestDevice(QObject):
```

Comment:
Removing all the Qubes OS-specific behavior from the export code. (:slightly_smiling_face:)